packages/uploadthing/package.json

```
@@ -10,7 +10,8 @@
     },
     "./server": {
       "import": "./dist/server.mjs",
-      "types": "./dist/server.d.ts"
+      "types": "./dist/server.d.ts",
+      "node": "./dist/server.mjs"
```

I think its better if we use the more generic `default`

```suggestion
      "default": "./dist/server.mjs"
```